berkeley-function-call-leaderboard/README.md

```diff
@@ -54,13 +62,14 @@ To run the executable test categories, there are 4 API keys to include:
     * Covid 19: https://rapidapi.com/api-sports/api/covid-193
     * Time zone by Location: https://rapidapi.com/BertoldVdb/api/timezone-by-location
 
-    All the Rapid APIs we use have free tier usage. As a result, you need to subscribe to those API providers in order to have the executable test environment setup but it will be free of charge!
+    All the Rapid APIs we use have free tier usage. You need to **subscribe** to those API providers in order to have the executable test environment setup but it will be *free of charge*!
 
 2. Exchange Rate API:https://www.exchangerate-api.com
 3. OMDB API: http://www.omdbapi.com/apikey.aspx
 4. Geocode API: https://geocode.maps.co/
 
 The `apply_function_credential_config.py` will automatically search for dataset files in the default `./data/` directory and replace the placeholder values with the actual API keys.
+After you have filled in the necessary values in the `.env` file, you can run the following command to apply the real API keys to the dataset files.
```

Comment:
IMO, if we have `.env`, we should change the wordings for this section in README
```
## Evaluating different models on the BFCL
Make sure the model API keys are included in your environment variables. Running proprietary models like GPTs, Claude, Mistral-X will require them.
```bash
export OPENAI_API_KEY=sk-XXXXXX
export MISTRAL_API_KEY=XXXXXX
export FIRE_WORKS_API_KEY=XXXXXX
export ANTHROPIC_API_KEY=XXXXXX
export COHERE_API_KEY=XXXXXX
export NVIDIA_API_KEY=nvapi-XXXXXX
export YI_API_KEY=XXXXXX


```

And only allow `.env` for passing environment variables for clarity and consistency.

Reply:
resolved